test/jdk/java/awt/TextArea/TextAreaCursorTest/HoveringAndDraggingTest.java

```diff
@@ -21,71 +21,76 @@
  * questions.
  */
 
-/*
-  test
-  @bug 6497109
-  @summary Mouse cursor icons for TextArea should be correct in case of hovering or dragging mouse over different subcomponents.
-  @author Konstantin Voloshin: area=awt.TextArea
-  @run applet/manual=yesno HoveringAndDraggingTest.html
-*/
-
-/**
- * HoveringAndDraggingTest.java
- *
- * summary: Mouse cursor icons for TextArea should be correct in case
- *   of hovering or dragging mouse over different subcomponents.
- */
-
 import java.awt.Frame;
 import java.awt.Panel;
 import java.awt.GridLayout;
 import java.awt.TextArea;
-import java.awt.Dialog;
 
-public class HoveringAndDraggingTest extends java.applet.Applet {
-    public void start() {
-        String[] instructions = new String[] {
-            "1. Notice components in test window: main-panel, box-for-text,"
-                +" 2 scroll-sliders, and 4 scroll-buttons.",
-            "2. Hover mouse over box-for-text."
-                +" Make sure, that mouse cursor is TextCursor (a.k.a. \"beam\").",
-            "3. Hover mouse over each of components (see item 1), except for box-for-text."
-                +" Make sure, that cursor is DefaultCursor (arrow).",
-            "4. Drag mouse (using any mouse button) from box-for-text to every"
-                +" component in item 1, and also outside application window."
-                +" Make sure, that cursor remains TextCursor while mouse button is pressed.",
-            "5. Repeat item 4 for each other component in item 1, except for box-for-text,"
-                +" _but_ now make sure that cursor is DefaultCursor.",
-            "6. If cursor behaves as described in items 2-3-4-5, then test passed; otherwise it failed."
-        };
-        Sysout.createDialogWithInstructions( instructions );
+/*
+ * @test
+ * @bug 6497109
+ * @summary Mouse cursor icons for TextArea should be correct in case of
+ *  hovering or dragging mouse over different subcomponents.
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual HoveringAndDraggingTest
+ */
+
+public class HoveringAndDraggingTest {
+    public static void main(String[] args) throws Exception {
+        String INSTRUCTIONS = """
+                    1. Notice components in test window: main-panel, box-for-text,
+                       2 scroll-sliders, and 4 scroll-buttons(Not applicable for macosx).
+                    2. Hover mouse over box-for-text.
+                       Make sure, that mouse cursor is TextCursor (a.k.a. \"beam\").
+                    3. Hover mouse over each of components (see item 1), except for box-for-text.
```

Comment:
I have updated column count to handle this scenario. It looks fine in my local machine, not sure how come its overflowing.
<img width="346" alt="image" src="https://github.com/user-attachments/assets/2a93c2b0-cecd-4cac-8cac-2183870de28f">

Reply:
I've encountered similar issues with my tests in the past where it looks fine locally. I chalked it up to different OS's, resolutions, text sizes, etc.

Now, I just set column to `40` in my test instructions and follow the standard guideline of `80` chars in my IDE to avoid these issues.